src/engine/ruleset/decoders/sophos/xg/sophos-antispam.yml

```diff
@@ -0,0 +1,127 @@
+---
+name: decoder/sophos-antispam/0
+
+metadata:
+  module: Sophos
+  title: Sophos-Antispam logs decoder
+  version: [17.5.x, 18.0.x]
+  description: Decoder for Sophos Antispam
+  author:
+    name: Wazuh Inc. info@wazuh.com
+    date: 2023-01-31
+  references:
+    - https://support.sophos.com/support/s/?language=en_US#t=DocumentationTab
+
+sources:
+  - decoder/queue-syslog/0
+  - decoder/queue-localfile/0
+
+check:
+  - event.original: +r_match/^device\=[[:alpha:]]*? date\=.*? time\=.*? timezone\=[[:alpha:]]*? device_name\=[[:alpha:]]*? device_id\=[[:alnum:]]*? log_id\=[[:alnum:]]*? log_type\="Anti-Spam"
```

Comment:
Check this regex. `[[:alpha:]]` does not match `"`
The same applies to all `xg` decoders.